### ISSUE TITLE
libobjc2, GNUstep Make and Base

### ISSUE THREAD

**Issue by trunkmaster** [trunkmaster/nextspace#127]
- [x] build libbobjc 2.0 library;
- [x] test libbobjc 2.0 library;
- [x] build GNUstep Make 2.7.0
- [x] build GNUstep Base 1.26.0

**Comment by trunkmaster** [comment]
libobjc-2.0 was successfully built with:
```shell
$ cmake3 .. -DGNUSTEP_INSTALL_TYPE=SYSTEM \
                  -DCMAKE_C_COMPILER=clang \
                  -DCMAKE_CXX_COMPILER=clang++ \
                  -DCMAKE_C_FLAGS=-I/usr/NextSpace/include \
                  -DCMAKE_LIBRARY_PATH=/usr/NextSpace/lib \
                  -DTESTS=OFF -DCMAKE_BUILD_TYPE=Release
$ make
```
All test are passed.
`$ make test`
[tests output skipped]
`Total Test time (real) =  40.82 sec`
`100% tests passed, 0 tests failed out of 162`

**Comment by trunkmaster** [comment]
My observation: Base does not function correctly with compiler flag `-fobjc-runtime=gnustep-2.0` specified.
With `-fobjc-runtime=gnustep-1.8` test results looks good. 

GNUstep Make automatically adds `-fblocks -fobjc-runtime=gnustep-1.8` to compiler options if it's built with `--with-library-combo=ng-gnu-gnu`. So the correct build sequence for GNUstep Make is (note: this sequence is good for creating RPM):
```shell
$ ./configure \
    --with-config-file=/Library/Preferences/GNUstep.conf \
    --with-layout=nextspace \
    --enable-native-objc-exceptions \
    --enable-debug-by-default \
    --with-library-combo=ng-gnu-gnu
$ make install
```

**Comment by trunkmaster** [comment]
GNUstep Base build succesfull. `make check` results:
```
8391 Passed tests
     26 Dashed hopes
      1 Skipped set
      1 Failed test
```
Failed test is `base/GSTLS`.
Skipped test is `base/NSRunLoop` with message:
```
dispatch.m 118 ... No libdispatch, no blocks support or no runloop integration hooks in libdispatch
```
Not ideal but quite good.

**Comment by trunkmaster** [comment]
Closed #127. Switched to #128.
